src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Standard/Verify.cs

```diff
@@ -59,17 +59,7 @@ public static void IsNeitherNullNorEmpty(string value, string name)
         {
             // catch caller errors, mixing up the parameters.  Name should never be empty.
             Assert.IsNeitherNullNorEmpty(name);
-
-            // Notice that ArgumentNullException and ArgumentException take the parameters in opposite order :P
-            const string errorMessage = "The parameter can not be either null or empty.";
-            if (null == value)
-            {
-                throw new ArgumentNullException(name, errorMessage);
-            }
-            if ("" == value)
-            {
-                throw new ArgumentException(errorMessage, name);
-            }
+            ArgumentException.ThrowIfNullOrEmpty(value, name);
```

Comment:
I'm very tempted to remove the assert above and inline this method the way I removed `IsNotNull` below. Should I go ahead and do that?

Reply:
My suggestion would be to create a separate pull request for that, so that this pull request can focus solely on the usage of ArgumentNullException or ArgumentException.

Reply:
I am not suggesting undoing the changes that have already been made, but I am suggesting that, create a separate PR for removing any the asserts.